enterprise/server/test/integration/remote_bazel/remote_bazel_test.go

```diff
@@ -140,7 +140,8 @@ func runRemoteBazelInSeparateProcess(t *testing.T, workDir string, serverAddress
 			"--runner_exec_properties=container-image=",
 		},
 		args...)...)
-	err := cmd.Run()
```

Comment:
Question I should have asked on the prior PR: does the CLI command have to run in a separate goroutine? Or is it okay for it to run to completion before proceeding with the test?

Reply:
Sorry don't totally understand your question. Are you asking whether the CLI command should run in the background? 

The rest of the test expects the CLI command to have completed running